src/html.sortable.src.js

```diff
@@ -192,6 +192,14 @@ var _reloadSortable = function(sortable) {
   // remove event handlers from sortable
   _removeSortableEvents(sortable);
 };
+/**
+ * Get position of the element relatively to its sibling elements
+ * @param {Element} element
+ * @returns {number}
+ */
+var _index = function(element) {
+  return [].indexOf.call(element.parentElement.children, element);
```

Comment:
I think you'd want `Array.prototype.indexOf` here as it does not create an empty array which has to be garbage collected.